run_pipeline.sh

```diff
@@ -1,72 +1,86 @@
 #!/bin/bash
-# Note, change the directory to where you want these things to appear before running this code.
+# C. Savonen
+# CCDL for ALSF 2018
 
-# Make directories
-mkdir data
-mkdir data/raw_data
-mkdir data/aligned_reads
-mkdir data/fastqc_reports
-mkdir data/fastqc_trimmed
-mkdir results
-mkdir results/map_qc
+# Purpose: running the pre-processing steps for single cell RNA-seq data.
 
-#--------------------------- Download fastq data-------------------------------#
-# Note: because running all ~3800 samples takes quite a bit of time, this 
-# example run is set to only run 10 samples. When you want to run the full set, 
-# delete the -n argument.  
-Rscript 0-download_fastq_data.R \
--i SRP079058 \
--d data/raw_data \
--n 20 
+# Change your directory name, GEO ID, and SRP here. Then run the script.
+directory=darmanis_data
+GSE=GSE84465
+SRP=SRP079058
 
-#------------------------Run fastqc and get reports----------------------------#
-# Run sequence quality control with FASTQC
-/FastQC/fastqc data/raw_data/* --outdir data/fastqc_reports
+#----------------------------Make directories---------------------------------#
+mkdir ${dir}
+mkdir ${dir}/raw_data
+mkdir ${dir}/aligned_reads
+mkdir ${dir}/fastqc_reports
+mkdir ${dir}/salmon_quants
+mkdir results
+mkdir ref_files
 
-# Obtain summary report:
-Rscript 1-get_fastqc_reports.R -d data/fastqc_reports -o results
+#--------------Will run setup only if it hasn't been ran before----------------#
+# Will check for genome index first before running
+if [ ! -e ref_files/human_index ]; then
 
-#--------------Paired trimming of the adapters from these data:----------------#
-# Go to the raw_data directory 
-cd data/raw_data
+  # Get the human transcriptome
+  curl ftp://ftp.ensembl.org/pub/release-94/fasta/homo_sapiens/cdna/Homo_sapiens.GRCh38.cdna.all.fa.gz \
+  -o ref_files/Homo_sapiens.GRCh38.cdna.all.fa.gz
 
-# Do adapter trimming for all sample file pairs
-for f in `ls *_1.fastq.gz | sed 's/_1.fastq.gz//'`
-do
-/TrimGalore-0.4.5/trim_galore \
---nextera \
---paired ${f}_1.fastq.gz ${f}_2.fastq.gz \
--o ../fastqc_trimmed
-done
+  # Index the human transcriptome
+  salmon --threads=16 --no-version-check index \
```

Comment:
I should have been more explicit. I'm expecting indentation like so:
```sh
  salmon --threads=16 --no-version-check index \
    -t ref_files/Homo_sapiens.GRCh38.cdna.all.fa.gz \
    -i ref_files/human_index \
    -k 23
```

That goes for the rest of the script as well.

Reply:
I see. I thought you were just talking about the loop. Gotcha. Will do that.

Reply:
Yea, my fault

Reply:
Did the indents. Anything else?